### PR TITLE
feat(myhpi): add info bit protobuf

### LINKED ISSUE #6: Add myhpi definitions

### PR DESCRIPTION
**Pull request by MarcelGarus**
**Fixes: #6**

**Checklist**

- [x] Commit messages follow conventionalcommits.org

**Changes**

### CODE REVIEW

File: hpi/cloud/myhpi/v1test/info_bit.proto [hunk 1]
```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package hpi.cloud.news.v1test;
+
+// A bit of information to show the user on the MyHPI screen.
```
Comment (JonasWanke) [inline hpi/cloud/myhpi/v1test/info_bit.proto]: This is too focused on the android app, please use a more generic description

File: hpi/cloud/myhpi/v1test/info_bit.proto [hunk 2]
```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package hpi.cloud.news.v1test;
```
Comment (JonasWanke) [inline hpi/cloud/myhpi/v1test/info_bit.proto]: Copy + Paste...

File: hpi/cloud/myhpi/v1test/info_bit.proto [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+// A bit of information to show the user on the MyHPI screen.
+message InfoBit {
+  // Required, output only. The unique ID of this info bit.
+  string id = 1;
+
+  // Required. The title.
+  string title = 2;
+
+  // Required. The text.
+  string text = 3;
```
Comment (JonasWanke) [inline hpi/cloud/myhpi/v1test/info_bit.proto]: Maybe `description` would be a better name?

**Review by JonasWanke — CHANGES_REQUESTED** [review]
The service definition is still missing.

**Comment by MarcelGarus** [comment]
> The service definition is still missing.

What's that about?

**Comment by JonasWanke** [comment]
> > The service definition is still missing.
> 
> What's that about?

Right now, only the data wrappers are defined. We still need a service (like [NewsService](https://github.com/HPI-de/hpi-cloud-apis/blob/dev/hpi/cloud/news/v1test/news_service.proto)) to list all methods available on the server

**Pull request closed without merging**